docs/content/docs/advanced/engine.md

```diff
@@ -3,15 +3,14 @@ title: Explanation about how the engine works
 incomplete: true
 ---
 
-During the build process, markdown files are parsed, and transformed as JSON
-files (one part with the YAML header meta informations, one part with the body
-as HTML).
-So both client and server can easily consume markdown files (as json) in order
-to render pages from React components. That's it !
+During the build process, markdown files are transformed into JSON; one part
+with the YAML metadata, and one part with the HTML body. So both client and
```

Comment:
since we are using https://www.npmjs.com/package/gray-matter I guess we should. Opening a new issue about that.